config/kubernetes/staging/grafana-applications.yml

```diff
@@ -499,7 +500,8 @@ data:
       "schemaVersion": 37,
       "style": "dark",
       "tags": [
-        "CrimeApply"
+        "CrimeApply",
+        "custom"
```

Comment:
Is this 'custom' tag required?

Reply:
No, you can add whatever tags you want. I added `custom` because this dashboard is build from custom metrics (with a custom collector on Apply).